src/events/GuildJoined.ts

```diff
@@ -15,6 +14,6 @@ export class GuildJoined extends Event
     {
         await registerGuildCommands(guild.id);
 
-        await sendLogMsg(`I just joined the guild "${guild.name}" and registered all slash commands there`);
+        // await sendLogMsg(`I just joined the guild "${guild.name}" and registered all slash commands there`);
```

Comment:
we might want it that's why I left it, so I can remember in 20 years when I look at this file again

Reply:
we'll need a different function since sendLogMsg() sends the stuff to the dedicated guild channel right? i suggest adding a TODO: that explains what needs to be done instead of commented code

Reply:
ok